usr/src/lib/libzfs/common/libzfs_pool.c

```diff
@@ -2425,6 +2425,7 @@ zpool_vdev_online(zpool_handle_t *zhp, const char *path, int flags,
 {
 	zfs_cmd_t zc = { 0 };
 	char msg[1024];
+	char *pathname = NULL;
```

Comment:
Is it necessary to initialise pathname in this case?

Reply:
Right, it isn't, and it wasn't necessary previously as well.